vars/mtaBuild.groovy

```diff
@@ -113,27 +124,33 @@ void call(Map parameters = [:]) {
                 echo "[INFO] '${mtaYamlName}' file found in project sources."
             }
 
-            def mtaYaml = readYaml file: mtaYamlName
-
             //[Q]: Why not yaml.dump()? [A]: This reformats the whole file.
             sh "sed -ie \"s/\\\${timestamp}/`date +%Y%m%d%H%M%S`/g\" \"${mtaYamlName}\""
 
-            def id = mtaYaml.ID
-            if (!id) {
-                error "Property 'ID' not found in ${mtaYamlName} file."
+            def id = getMtaId(mtaYamlName)
+
+            def mtaCall
+            def options = []
+            options.push("--mtar ${id}.mtar")
+
+
+            switch(configuration.buildTool) {
+                case 'classic':
+                    // If it is not configured, it is expected on the PATH
+                    def mtaJar = configuration.mtaJarLocation ?: 'mta.jar'
+                    options.push("--build-target=${configuration.buildTarget}")
+                    if (configuration.extension) options.push("--extension=${configuration.extension}")
+                    mtaCall = "java -jar ${mtaJar} ${options.join(' ')} build"
+                    break
+                case 'cloudMBT':
```

Comment:
minor: I dislike the way of spelling, something like `cloudMbt` looks nicer imo

Reply:
fine with me